### PR TITLE
Bump stencil core to 2.16.0 and use experimentalImportInjection

### PR DESCRIPTION
**Pull request by Stiffni**
No tasks associated

### Summary of changes included in this PR

- Update stencil to [v2.16.0](https://github.com/ionic-team/stencil/releases/tag/v2.16.0) and add `extras.experimentalImportInjection = true` to stencil config to fix compatibility with vite projects

### Reviewer checklist:

- Tests for
  - accessibility
  - rendered HTML (spec)
- Storybook stories for all variants
- JSDoc documentation for component
- Update Example app (React, ...)

### CODE REVIEW

**Comment by Stiffni** [comment]
@TanguyDucky We tried it with ducky-web locally 😏 as long as that version of `@ducky/plumage` in `component-library-react` auto-updates to the latest plumage version this should work 👍 (Johannes helped me get all that working locally)

**Comment by TanguyDucky** [comment]
It will update automatically, yes :grin: 
And it worked well? That's super cool!

**Review by TanguyDucky — APPROVED** [review]

**Comment by TanguyDucky** [comment]
Thanks! I've added a new example app using react and vite to demonstrate the use case.

**Review by TanguyDucky — APPROVED** [review]

File: common/changes/@ducky/plumage/build-update-stencil-core_2022-06-01-18-16.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@ducky/plumage",
+      "comment": "Bump Stencil version to 2.16.0, add experimentalImportInjection flag to stencil.config.ts",
+      "type": "patch"
```
Comment (TanguyDucky) [inline common/changes/@ducky/plumage/build-update-stencil-core_2022-06-01-18-16.json:6]:
```suggestion
      "type": "minor"
```

**Review by TanguyDucky — APPROVED** [review]

**Pull request merged**